commitment.go

```diff
@@ -19,6 +19,16 @@ var ErrCommitmentMismatch = errors.New("commitment mismatch")
 // CommitmentType is the commitment type prefix.
 type CommitmentType byte
 
+// Max input size ensures the canonical chain cannot include input batches too large to
+// challenge in the Data Availability Challenge contract. Value in number of bytes.
+// This value can only be changed in a hard fork.
+const MaxInputSize = 130672
```

Comment:
This is just a sanity check right? I would think OP Stack should be enforcing this invariant.

Reply:
You are correct - left over copy pasta

Reply:
[a28ca7e](https://github.com/Layr-Labs/op-plasma-eigenda/pull/8/commits/a28ca7efd1cb66276ac9d0d20cf605a1f43a9b01)